src/lib/y2firstboot/clients/licenses.rb

```diff
@@ -53,7 +53,7 @@ def run
       def inst_license_args
         args = GetInstArgs.argmap
         args["action"] = refusal_action
-        args["directories"] = licenses_directories
+        args["directories"] = licenses_directories if args["directory"].to_s.empty?
```

Comment:
As already commented offline, I would prefer to continue using `directories` parameter, but with the proper license paths for each case: directory path from the module or directory paths from the sysconfig file. This will allow to show several module licenses in the same page (in case someone requests that in the future).

Reply:
Done. See the new commit.